synapse/lib/layer.py

```diff
@@ -1451,25 +1451,29 @@ async def _layrV5toV6(self):
         self.meta.set('version', 6)
         self.layrvers = 6
 
-        logger.warning(f'...complete!')
+        logger.warning('...complete!')
 
     async def _initLayerStorage(self):
 
         slabopts = {
             'readonly': self.readonly,
             'max_dbs': 128,
-            'map_async': True,
             'readahead': True,
             'lockmemory': self.lockmemory,
-            'growsize': self.growsize,
+            'map_async': self.mapasync,
         }
 
+        if self.growsize is not None:
+            slabopts['growsize'] = self.growsize
+
+        otherslabopts = slabopts.copy()
+        otherslabopts['readahead'] = False
```

Comment:
Why is this the new default and what effect will it have on existing deployments?

Reply:
Ultimately this changes the nodeeditsslab mapasync to be consistent with the user setting (usually no changes), and turns off readahead for the nodeeditslab, which should slightly reduce I/O overhead for that. These are changes we probably wanted.

Restructured to be a bit more clear.